app/[lang]/(inferenceai)/inferenceai/rag-chatbot/data.tsx

```diff
@@ -48,7 +48,24 @@ import {
 
 export const data = {
   name: "RAG Chatbot",
-  baseUrl: "https://hyperjump.tech/inferenceai/rag-chatbot"
+  baseUrl: "https://hyperjump.tech/inferenceai/rag-chatbot",
+  socials: [
```

Comment:
You can import the socials data from [/app/[lang]/(inferenceai)/inferenceai/data.ts](https://github.com/hyperjumptech/hyperjump.tech/blob/d264fda2cc21aaa5b55cae818c1b2ac7a171acd6/app/%5Blang%5D/(inferenceai)/inferenceai/data.ts#L180).